app/models/instrument.rb

```diff
@@ -31,6 +31,13 @@ class Instrument < Product
 
   after_create :set_default_pricing
 
+  # Scopes
+  # --------
+  scope :reservation_only, lambda {
```

Comment:
At least for new code. There's a lot of old code that we haven't felt has been worth the effort to update.